functorch/functorch/_src/aot_autograd.py

```diff
@@ -277,8 +278,26 @@ def new_fn(args):
     return new_fn
 
 
+pre_autograd_decompositions = get_decompositions(
```

Comment:
@ngimel 
I couldn't place this in _scr/compilers.py due to circular module import... Keep this in current file.